### ISSUE TITLE
Missing docstring for isfishercons and isunivfishercons

### ISSUE THREAD

**Issue by juliohm** [JuliaML/LearnBase.jl#39]
These two traits exist for margin-based losses, but I couldn't find their docstrings in LossFunctions.jl to port over here. @Evizero do you have some references that you could share?

**Comment by juliohm** [comment]
Dear @Evizero, I know you are not involved with Julia development lately, but any chance you have some reference on this subject? I'd be happy to fix the issue myself, but I am currently lacking the references that you used when the trait was introduced in the package.

Sincerely,
-Julio

**Comment by Evizero** [comment]
Hey! Sorry I missed these mentions. I am rarely checking my github notifications anymore.

Mhm it has been a long while since i worked on losses but I think to recall the basic reasoning

Some context: When I started on LossFunctions.jl my primary interest was on support vector machines (I just got carried away with the subtopic of loss functions). Steinwart and Christmann wrote a book on the subject called "Support Vector Machines", which dedicates around 80 pages to loss functions . Most of the LossFunctions.jl package design (including the properties) was based on said book.

While diving into the subject of loss functions, I noticed there are additional properties discussed in other literature and I simply tried to be as featureful as I could. I don't recall where I exactly i got the definitions for fisher consistency from but this PDF looks about right: https://www.asc.ohio-state.edu/lee.2272/881/consistency.pdf . This paper also list the definitions https://www.researchgate.net/publication/23936205_New_multicategory_boosting_algorithms_based_on_multicategory_Fisher-consistent_losses including the universal case.

Note though that I never actually had a usecase for the property. I just got carried away in my eagerness to write a cool and complete package.

Hope this helps. 

Thank you for all your efforts.
Chris
